src/pages/Login.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react'
+import GoogleLogin from 'react-google-login'
+import '../App.css'
+
+function Login() {
+  const GOOGLE_CLIENT_ID = process.env.REACT_APP_CLIENT_ID || ''
+  const [user, setUser] = useState({ email: '', name: '' })
+  const onLoginSuccess = (res: any) => {
+    // res type 정의 필요
+    setUser({ ...user, email: res.profileObj.email, name: res.profileObj.name })
+  }
+  return (
+    <div className="App">
+      <header className="App-header">
+        <GoogleLogin
+          clientId={GOOGLE_CLIENT_ID}
+          buttonText="Google Login"
+          onSuccess={result => onLoginSuccess(result)}
+          onFailure={result => console.log(result)}
```

Comment:
컨벤션을 정해야할것같은데 위에 function을 무조건 만드는 방식이 좋을까요?
그리고 아래처럼 수정도 가능합니다
```suggestion
          onSuccess={onLoginSuccess}
          onFailure={console.log}
```